service/experiment-requests.yaml

```diff
@@ -19,6 +19,30 @@
 # You can run "make presubmit" to do basic validation on this file.
 # Please add new experiment requests towards the top of this file.
 #
+- experiment: 2022-09-27-centipede-code-saturated
```

Comment:
Not sure if this long experiment name causes failure.

Reply:
yep, thanks :) i already fixed this in https://github.com/google/fuzzbench/pull/1500